src/main/java/org/truffleruby/core/string/StringNodes.java

```diff
@@ -517,8 +517,8 @@ public Object sliceObjectRange(VirtualFrame frame, DynamicObject string, Dynamic
                 @Cached("new()") SnippetNode snippetNode1,
                 @Cached("new()") SnippetNode snippetNode2) {
             // TODO (nirvdrum 31-Mar-15) The begin and end values may return Fixnums beyond int boundaries and we should handle that -- Bignums are always errors.
-            final int coercedBegin = (int) snippetNode1.execute(frame, "Rubinius::Type.rb_num2int(v)", "v", Layouts.OBJECT_RANGE.getBegin(range));
-            final int coercedEnd = (int) snippetNode2.execute(frame, "Rubinius::Type.rb_num2int(v)", "v", Layouts.OBJECT_RANGE.getEnd(range));
+            final int coercedBegin = (int) snippetNode1.execute(frame, "Truffle::Support::Type.rb_num2int(v)", "v", Layouts.OBJECT_RANGE.getBegin(range));
```

Comment:
Can it be just `Truffle::Type`/

Reply:
You're asking the same question for all of these...

Reply:
Yes, I just wanted to give examples of the important ones we should simplify